pytest_dbt_adapter/spec_file.py

```diff
@@ -323,8 +335,8 @@ def step_run_results(self, sequence_item, tmpdir):
 
             for result in results:
                 try:
-                    node = result['node']
-                    name = node['name']
+                    unique_id = result['unique_id']
```

Comment:
Ideally we could write this in a way that works for pre-0.19 versions, too, with handling for either `result.get('unique_id')` or `result.get('node', {}).get('unique_id')`.